### DISCUSSION TITLE
Teleport

### DISCUSSION THREAD

**Discussion by cfragkos** [coollabsio/coolify#1637] [Service Template Requests]
Add [teleport](https://goteleport.com/) as a service

**Comment by compgeniuses** [comment]
this used to be opensource, its seems its not opensource anymore or is it, and where is the docker compose?

**Reply by marko-hologram** [reply]
I don't think they ever had compose file listed in their docs. If I remember correctly, it was always documented using "docker run" there.
https://goteleport.com/docs/installation/#docker

And it seems there is still a self hosted community edition available
![image](https://github.com/coollabsio/coolify/assets/22904204/05aa2195-4258-4bf0-9482-1c380161a422)

**Reply by compgeniuses** [reply]
yes i noticed this.

I have been looking for a well structured doc or detail, something more similar to docker compose for deployment

**Reply by marko-hologram** [reply]
I think there is no official example that uses docker compose, but there are tutorials online such as this one https://www.youtube.com/watch?v=NzSdNoR-JPo

It's a more manual setup not related to Coolify, but definitely a good source of information.

**Reply by compgeniuses** [reply]
someone once said, developers dont read documentations, they read code to figure out whats happening....
